### ISSUE TITLE
False positive no-unsafe-references

### ISSUE THREAD

**Issue by creage** [playwright-community/eslint-plugin-playwright#213]
That new `no-unsafe-references` rule fires errors for all of native refs, like
```
Unsafe reference to variable "Promise" in page.evaluate()
Unsafe reference to variable "window" in page.evaluate()
Unsafe reference to variable "JSON" in page.evaluate()
```

**Comment by mskelton** [comment]
@creage I just kicked off a release to fix this bug. Thanks for the report.

**Comment by creage** [comment]
Works fine on latest release. That was fast - thank you!
